guides/common/modules/proc_configuring-a-host-to-use-the-pull-client.adoc

```diff
@@ -47,13 +56,53 @@ ifdef::client-content-zypper[]
 ----
 endif::[]
 
+.Verification
+. Determine which version of the `yggdrasil` package is installed on the host:
+ifdef::client-content-dnf[]
+* On {EL} 9 and {EL} 8 hosts:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+$ dnf list installed yggdrasil
+----
+endif::[]
+ifdef::orcharhino,satellite[]
+* On {EL} 7 hosts:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+$ yum list installed yggdrasil
+----
+endif::[]
+ifdef::client-content-apt[]
+* On {DL} hosts:
 +
-The package installs `foreman_ygg_worker` and `yggdrasil` as dependencies, configures the `yggdrasil` client, and starts the pull client worker on the host.
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+$ apt list --installed | grep yggdrasil
```

Comment:
Same as the rpm -q comment above, on debian something like `dpkg-query --show $package` would be an alternative, but I'm not a debian person so feel free to ignore this.

Reply:
Seems legit, applied (here and elsewhere).